tools/autograd/gen_python_functions.py

```diff
@@ -225,6 +228,10 @@ def signature_original(f: NativeFunction) -> str:
         opname = str(f.func.name.name.base)
         if f.func.is_out_fn():
             opname += '_out'
+        # TODO: remove HACK
+        # I think we want to differentiate inplace functions here.. but we currently don't for the arg parser
+        if f.func.name.name.inplace and pyi:
+            opname += '_'
```

Comment:
This is causing some deprecated ops to show up in the python arg parser where they probably shouldn't be. E.g. `addmv_` is not actually a deprecated function.

Before:
```
 // addmv_
 static PyObject * THPVariable_addmv_(PyObject* self_, PyObject* args, PyObject* kwargs)
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
     "addmv_(Scalar beta, Tensor input, Scalar alpha, Tensor mat, Tensor vec)|deprecated",
     "addmv_(Scalar beta, Tensor input, Tensor mat, Tensor vec)|deprecated",
     "addmv_(Tensor input, Tensor mat, Tensor vec, *, Scalar beta=1, Scalar alpha=1)",
   }, /*traceable=*/true);
```

After:
```
 // addmv_
 static PyObject * THPVariable_addmv_(PyObject* self_, PyObject* args, PyObject* kwargs)
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
     "addmv_(Tensor input, Tensor mat, Tensor vec, *, Scalar beta=1, Scalar alpha=1)",
   }, /*traceable=*/true);
```

Reply:
Actually, I think the application of `addmv_` here is intentional. I went spelunking to de757805fcdbcbb831d51827d72e73e55a49a106 where deprecated.yaml was added, and here are the Python signatures it was replacing:

```
    def addmv(self, *args):
        return self._blas(Addmv, args, False)
        
    def addmv_(self, *args):
        return self._blas(Addmv, args, True)
```

it seems likely to me that the functional and inplace variants took the same legacy signatures.

Speaking of legacy, these signatures are soooo old, I wonder if we should get rid of them (not in this PR, of course).